src/components/dataset/show.vue

```diff
@@ -0,0 +1,124 @@
+<!--
+Copyright 2023 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+
+<template>
+  <div id="dataset-show">
+    <page-back v-show="dataExists" :to="projectPath('datasets')">
+      <template #title>{{ project.dataExists ? project.nameWithArchived : '' }}</template>
+      <template #back>{{ $t('back') }}</template>
+    </page-back>
```

Comment:
Could you add tests of the back link?

- The link points to the right page. (You can just check the `to` prop of the `PageBack` component. Check out the parallel test in `SubmissionShow` for inspiration.)
- The back link show's the project's name. (`FormHead` has a parallel test.)
- The back link appends "(archived)" to the project's name if the project is archived. (`FormHead` has a parallel test.)